src/com/carolinarollergirls/scoreboard/jetty/WS.java

```diff
@@ -28,15 +28,17 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import com.carolinarollergirls.scoreboard.ScoreBoard;
 import com.carolinarollergirls.scoreboard.ScoreBoardManager;
 import com.carolinarollergirls.scoreboard.json.ScoreBoardJSONListener;
+import com.carolinarollergirls.scoreboard.json.WSUpdate;
 import com.carolinarollergirls.scoreboard.model.ScoreBoardModel;
 
 public class WS extends WebSocketServlet {
+	private static final long serialVersionUID = 1L;
```

Comment:
This is unused

Reply:
Not having it in a serializable class will generate a warning, even if we don't serialize.

Reply:
We don't have this in any other class. Where are you seeing these warnings?

Also we shouldn't be using Java serialisation anywhere.

Reply:
Eclipse gives the warning for 17 files that all implement java.io.Serializable (inherited through some other interface).

Reply:
I don't think we should be cluttering the code due to false-positive warnings from one editor.

Reply:
The regular javac used by ant also gives this warning (if you don't configure ant to hide all warnings).

Reply:
I see no warnings from the default build, Java 1.8.0_162-8u162-b12-0ubuntu0.16.04.2-b12. If there are such warnings we should disable them.

Reply:
Setting nowarn to false and xlint:all in build.xml gives 99 warnings with javac 1.8.0_172
And I'm against disabling warnings. If these classes don't need to be serializable, don't implement that interface. But if you implement it, do it properly. (That being said: using 1 as UID is also improper. )

Reply:
So it's not a warning, it's a lint. If we're going to enable those we should do so as a separate PR, not piecemeal. We should also make sure that any warnings are actually issues, and not false positives like this one.

Serialisation is a very old java feature that noone should be using these days, I don't think our code should suffer due to its legacy.

Reply:
And looks like it's easy to disable that particular lint warning.


I'm not against useful warnings, but I also like reasonably self-contained changes.